### PR TITLE
Add base package with lerna, update README

### PR DESCRIPTION
**Pull request by draffensperger**
This use of `lerna` parallels the [OpenCensus node](https://github.com/census-instrumentation/opencensus-node) use of it.

Having the web specific packages in a separate repo I think makes sense as it allows cleaner separation of Node vs. web functionality, and we can run separate test suites for each one. The `@opencensus/core` interfaces at least will be used in both places though so if it would be better to combine we can.

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.11.0",
```
Comment (mayurkale22) [inline lerna.json]: Can we use the latest version here? https://www.npmjs.com/package/lerna

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "test": "lerna run test",
+    "bootstrap": "lerna bootstrap",
+    "bump": "lerna publish",
+    "codecov": "lerna run codecov"
+  },
+  "keywords": [
+    "opencensus",
+    "browser",
+    "tracing",
+    "profiling"
```
Comment (mayurkale22) [inline package.json]: We should add ```Stats``` and ```Metrics``` here, I am gonna do the same for OC-Node repo.

File: lerna.json (hunk 1 above)
Comment (justindsmith) [reply lerna.json]: We pull in `"lerna": "^3.7.1"` in the devDependencies. We should unify these at the very least, right?

File: package.json (hunk 2 above)
Comment (draffensperger) [reply package.json]: Done.

File: lerna.json (hunk 1 above)
Comment (draffensperger) [reply lerna.json]: I switched this to the latest version and also started a PR to upgrade it for `opencensus-node` as well: https://github.com/census-instrumentation/opencensus-node/pull/263

Comment (draffensperger) [reply lerna.json]: Realized I misunderstood the comment about unification that this involve fixing `lerna.json` here not necessarily upgrading it in `opencensus-node`. Fixed it here to match the `3.8.5` in `devDependencies`.

**Comment by draffensperger** [comment]
@mayurkale22 does this look good? If so, can you approve?

**Pull request merged**